### ISSUE TITLE
Question about data pre-processing

### ISSUE THREAD

**Issue by xlk369293141** [TemporalKGTeam/TANGO#7]
First of all, thank you for sharing.

I am working on the future link prediction of TKG. I would like to reproduce the results of 4 other datasets and compare them, is it possible to share the rest of the datasets and their preprocessing code? 

I notice that mapping.py recombines the training set, the validation set and the test set using the following code. 
 `    
len_train, len_val = int(file.shape[0] * 0.8)+41, int(file.shape[0] * 0.1)+56
`
Do the other datasets also use this segmentation criteria?

Looking forward to your reply, thanks!
